relay/relay.go

```diff
@@ -77,7 +84,9 @@ func (r *Relay) Start(ctx context.Context) error {
 		client.Start(ctx)
 	}
 
-	recentFeedItems := make(map[arbutil.MessageIndex]time.Time)
+	var lastConfirmed arbutil.MessageIndex
+	recentFeedItemsNew := make(map[arbutil.MessageIndex]time.Time, RECENT_FEED_INITIAL_MAP_SIZE)
+	recentFeedItemsOld := make(map[arbutil.MessageIndex]time.Time, RECENT_FEED_INITIAL_MAP_SIZE)
```

Comment:
Perhaps we could use an LRU cache instead for simplicity? I'm introducing a generic one in https://github.com/OffchainLabs/nitro/pull/1175 but we shouldn't block this PR on that one. This approach works too though.

Reply:
having two buckets is more granular than an LRU cache, but I like that it never has to do any reordering.